app/locales/en/translation.json

```diff
@@ -30,7 +30,9 @@
     "reallyDeleteAdvisory": "Really delete this advisory? This action cannot be undone.",
     "uploadFromFilesystem": "Upload from filesystem",
     "useTemplate": "Use a template",
-    "examples": "Examples"
+    "examples": "Examples",
+    "collapsibleHide": "hide",
+    "collapsibleShow": "show"
```

Comment:
Aren't we missing the `base inputs`, `temporal inputs` and `environmental inputs` here? (or down in the CVSS Section?)

Reply:
Yes, indeed missing. I added those in a separate section in the translation file for cvss editors.